src/aux-runtime/runtime/AuxLibrary.spec.ts

```diff
@@ -17031,6 +17031,23 @@ describe('AuxLibrary', () => {
                 )
             ).toBe('AQIDBAU=');
         });
+
+        it('should convert the given value to bytes when given an ArrayBuffer', () => {
+            const base64String = 'AQIDBAU=';
+            const arrayBuffer =
+                library.api.bytes.fromBase64String(base64String);
```

Comment:
@TroyceGowdy This should test `bytes.toBase64String()` instead of `bytes.fromBase64String()`